allspice/apiobject.py

```diff
@@ -1219,6 +1345,30 @@ def delete(self):
 
 
 class Milestone(ApiObject):
+    allow_merge_commits: Any
```

Comment:
hmm, do you know why these are almost all `Any` type?

Reply:
It's explained here: https://github.com/AllSpiceIO/py-allspice/pull/141/files#diff-309fedefaae97b5dd4967e2a97571b17a8fcb1f7991b2d9bcb13397bd041921aR282-R287

Basically, we don't have a test where these values are set, and `Any` is better than `None` here.

Reply:
Got it. Thanks 👍 
Kind of doubles as code-coverage 🤔